File: app/src/main/java/co/wm21/https/FHelper/networks/Models/application/BDPStatusData.java
package co.wm21.https.FHelper.networks.Models.application;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class BDPStatusData {
    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("user")
    @Expose
    private String user;
    @SerializedName("amount")
    @Expose
    private String amount;
    @SerializedName("unite")
    @Expose
    private String unite;
    @SerializedName("date")
    @Expose
    private String date;
    @SerializedName("approve")
    @Expose
    private String approve;
    @SerializedName("paid")
    @Expose
    private String paid;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public String getUnite() {
        return unite;
    }

    public void setUnite(String unite) {
        this.unite = unite;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getApprove() {
        return approve;
    }

    public void setApprove(String approve) {
        this.approve = approve;
    }

    public String getPaid() {
        return paid;
    }

    public void setPaid(String paid) {
        this.paid = paid;
    }
}

File: app/src/main/java/co/wm21/https/FHelper/networks/Models/application/BDPStatusData.kt
package co.wm21.https.FHelper.networks.Models.application

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class BDPStatusData {
    @JvmField
    @SerializedName("id")
    @Expose
    var id: String? = null

    @JvmField
    @SerializedName("user")
    @Expose
    var user: String? = null

    @JvmField
    @SerializedName("amount")
    @Expose
    var amount: String? = null

    @JvmField
    @SerializedName("unite")
    @Expose
    var unite: String? = null

    @JvmField
    @SerializedName("date")
    @Expose
    var date: String? = null

    @JvmField
    @SerializedName("approve")
    @Expose
    var approve: String? = null

    @JvmField
    @SerializedName("paid")
    @Expose
    var paid: String? = null
}
